docs/modules/kubernetes/pages/helm-upgrade-guide.adoc

```diff
@@ -0,0 +1,140 @@
+= Helm Upgrade Guide
+
+Before upgrade, we recommend you to check link:https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rolling-upgrades#hazelcast-members-compatibility-guarantees[Hazelcast's Compatibility Guarantees].
+
+If you are not using hotRestart (aka persistence) feature and Management Center's persistence feature, tests cannot cover every scenario, but no problems expected during upgrade.
```

Comment:
I think this note makes it a bit confusing, because we cannot cover every scenario in case persistence is enabled, too. From the user's perspective, this sentence makes me think they cover all scenarios when persistence is enabled, but they cannot cover when it is disabled.